extras/lavinmq.ini

```
@@ -22,3 +40,25 @@ tcp_proxy_protocol = false
 tls_port = 5671
 unix_path = /tmp/lavinmq.sock
 unix_proxy_protocol = true
+;channel_max = 2048
+;frame_max = 1048576
+;heartbeat = 0
+;max_message_size = 128 * 1024**2
```

can't be an expression, have to a fixed number
```suggestion
;max_message_size = 134217728
```